Cargo.toml

```diff
@@ -8,6 +8,6 @@ license = "BSD-3-Clause"
 description = "ASN.1 (DER) serializer for Rust."
 
 [dependencies]
-byteorder = "*"
+byteorder = ">= 0.5.0"
```

Comment:
I'm probably not going to burn that bridge until I get to it, my experience
has been that encouraging forward progress (as this does) is generally
correct.

On Sat, Apr 2, 2016 at 5:35 PM, Danilo Bargen <notifications@github.com>
wrote:

> In Cargo.toml
> <https://github.com/alex/rust-asn1/pull/45#discussion_r58300259>:
>
> > @@ -8,6 +8,6 @@ license = "BSD-3-Clause"
> >  description = "ASN.1 (DER) serializer for Rust."
> >
> >  [dependencies]
> > -byteorder = "*"
> > +byteorder = ">= 0.5.0"
>
> This way, an incompatible release 0.6.0 or 1.0.0 would break rust-asn1.
> Version pinning (to 0.5.x) is less of a problem than a project not building
> anymore.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/alex/rust-asn1/pull/45/files/e404c22776597d9bf2b2b27bea98decafe451e29#r58300259>
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6